ctk/common/src/main/kotlin/org/codice/compliance/verification/profile/SingleSignOnProfileVerifier.kt

```diff
@@ -60,6 +61,7 @@ class SingleSignOnProfileVerifier(private val authnRequest: AuthnRequest,
         verifySsoAssertions()
         SubjectComparisonVerifier(samlResponseDom).verifySubjectsMatchSSO()
         ProfilesVerifier(samlResponseDom).verify()
+        PostBindingVerifier().verifyPostSSO(samlResponseDom)
```

Comment:
Not sure why this is related to Artifact, it's just verifying that if it's POST the response or each assertion has to be signed. Another thing to keep in mind is that redirect is an allowed response binding for SLO. 
I'm not in favor of having every test call this method individually. The verification is still in the Binding verifier. It's just a profile specific verification.

Reply:
My point is that, by calling this verifier in the SSOProfileVerifier, you're assuming the SSO response is POST binding. It could be Artifact binding in the future if we add support for it. At that point, this verification call shouldn't be called for all SSO tests. Only ones that get a POST response.

Reply:
Ohhhh